pyproject.toml

```diff
@@ -64,13 +64,7 @@ module = [
 ]
 ignore_missing_imports = true
 
-
-
-[tool.black]
-target-version = ["py39", "py310", "py311"]
```

Comment:
this is not needed since black 23.1.0, see https://black.readthedocs.io/en/stable/change_log.html#id85